src/lib/components/workflow/search-attribute-input/index.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+  import * as dateTz from 'date-fns-tz';
+
+  import Button from '$lib/holocene/button.svelte';
+  import Input from '$lib/holocene/input/input.svelte';
+  import Option from '$lib/holocene/select/option.svelte';
+  import Select from '$lib/holocene/select/select.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import {
+    customSearchAttributes,
+    type SearchAttributeInput,
+    type SearchAttributeInputValue,
+  } from '$lib/stores/search-attributes';
+  import type { SearchAttributes } from '$lib/types/workflows';
+
+  import NumberInput from './number-input.svelte';
+  import TextInput from './text-input.svelte';
+
+  export let attributesToAdd: SearchAttributeInput[] = [];
+  export let searchAttributes: SearchAttributes = $customSearchAttributes;
+  export let attribute: SearchAttributeInput;
+  export let onRemove: (attribute: string) => void;
+
+  $: type = searchAttributes[attribute.attribute];
+  $: searchAttributesOptions = [...Object.entries(searchAttributes)]
+    .map(([key, value]) => ({ label: key, value: key, type: value }))
+    .filter(({ type }) => type !== 'KeywordList');
```

Comment:
Do we want to support `KeywordList` attributes here? Not sure what that would look like, maybe a [ChipInput](https://holocene.preview.thundergun.io/?path=/story/chip-input--default)?

Reply:
I think we'll skip KeywordList for now and do a full update of that everywhere when ready including the Workflow Search UI